src/main/java/mate/academy/spring/Main.java

```diff
@@ -1,4 +1,20 @@
 package mate.academy.spring;
 
+import mate.academy.spring.config.AppConfig;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.UserService;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
 public class Main {
+    private static final AnnotationConfigApplicationContext context =
+            new AnnotationConfigApplicationContext(AppConfig.class);
+
+    public static void main(String[] args) {
+        User danylo = new User();
+        danylo.setEmail("danylo@gmail.com");
+        danylo.setPassword("111");
+        UserService userService = context.getBean(UserService.class);
```

Comment:
u can also make it as a field of Main class

Reply:
yeah, I cannnnnnnn